python/doc/examples/meta_modeling/general_purpose_metamodels/plot_distribution_linear_regression.py

```diff
@@ -138,7 +138,10 @@ def plot_sample_by_kernel_smoothing(
     bb = graph.getBoundingBox()
     ylb = bb.getLowerBound()[1]
     yub = bb.getUpperBound()[1]
-    curve = ot.Curve([true_standard_deviation**2] * 2, [ylb, yub])
+    if estimator == "variance": 
```

Comment:
+1 This explains why the last figure seems to be particularily wrong:

![image](https://github.com/user-attachments/assets/adef32ca-51c5-4ba0-b2fe-b97f707b58bb)

The vertical line is at $\sigma^2$, instead of $\sigma$. Thank you for solving this issue.

Reply:
yes...